### ISSUE TITLE
Any way to set preserveAspectRatio?

### ISSUE THREAD

**Issue by tjl9tq** [LottieFiles/lottie-react#19]
preserveAspectRatio on the svg seems to default to `xMidyMid meet` and there doesn't seem to be any way to change it. (Or at least i can't find any way). Could a way to set it be added?

**Comment by ajbater** [comment]
I can't find any way to do this either, can anyone help?

**Comment by tjl9tq** [comment]
So the way I ended up doing it, is I directly went into the dom after mounting the component with document selectors and manually changed the attribute myself. Not the cleanest solution, but the best I could come up with

**Comment by ajbater** [comment]
Thanks @tjl9tq I guess I will do the same!

**Comment by karamalie** [comment]
Will add this to the task list

Thanks for the feedback!

**Comment by karamalie** [comment]
Pushed version 3.0.1 to the npm package. 

prop rendererSettings has been added and you may add it as follows

```
 <Player
        src="https://assets6.lottiefiles.com/packages/lf20_V9t630.json"
        autoplay
        loop
        background="#ffffff"
        style={{ height: '300px' }}
        renderer="svg"
        onEvent={onEvent}
        rendererSettings={{
                                          clearCanvas: false,
                                          hideOnTransparent: true,
                                          progressiveLoad: true,
                                          preservereAspectRatio: 'none'
                                        }}
      >
 </Player>
```

Hope this helps :)

**Comment by ajbater** [comment]
Amazing, this is really helpful, thanks!

**Comment by consolenaut** [comment]
Hey! We just ran into this too and its great to hear there's a prop to do it now! 

Has 3.0.1 been published to NPM? I can only see 3.0.0 as the latest from about a month ago on the @lottiefiles/react-lottie-player module (https://www.npmjs.com/package/@lottiefiles/react-lottie-player)
